hack/repos.sh

```diff
@@ -35,6 +35,7 @@ repos=(
     cri-api
     csi-translation-lib
     dynamic-resource-allocation
+    endpointslice
```

Comment:
@mrbobbytables Dont we need a corresponding change in the rules file in k/k also, so that the repo gets published.